### ISSUE TITLE
Move to Zulip from Slack

### ISSUE THREAD

**Issue by allyunion** [ProgressiveCoders/functions#203]
#### Description

Move to Zulip.org from Slack

#### Problem

Slack has to be paid per user for searching and logging, Zulip will offer this free to ProgCode as an indefinite beta + hosting in exchange for feedback and bug reports, plus there is a Slack integration to help us migrate

#### Benefit

We need a communication tool that is as functional as Slack that allows us to view the logging chat history

#### Plan

- Create a Zulip account [jylee or someone else]
- Ensure administrative access to Zulip and Slack to tool migrators
- Migrate and export all users
- Inform all users of the migration and ask for volunteer testers
- Test with small set of users (alpha users)
- Get secondary approval with alpha users
- Full user migration

#### Decision Making

Initial consent to approval: Operations or Board Members for experiment
Secondary consent to approval: Community approval with alpha user tests

#### Reference link(s)

www.zulip,org

**Comment by patcon** [comment]
i recommend the carrot approach to comms tool migration -- don't force anything, but simply get permission to bridge with another chat tool, and see who follows. Prepare for a scenario where you're maintaining the bridge indefinitely, should no political will for full migration ever materialize.

In short, I don't think aggressive migration (ie. vote and force) is warranted -- love the UX of Slack, and that it comes across as very accessible to non-technical persons, many of whom are already using it in their own organizations.

**Comment by patcon** [comment]
Would be more supportive of an exploratory proposal to compare existing tools. Has that been done?

**Comment by jpb5013** [comment]
Agreed with @patcon, trying to force migration to another solution is a quick way to kill the community. There's unfortunately too much to lose from the platform monopoly, though an interoperable bridge is valid. I'd add that as a starting place could be introducing a separate message board (Zulip or other open source alternative like Matter Most or Rocket Chat) as an archive for people to access old messages.

**Comment by patcon** [comment]
fwiw, g0v movement (an anarchic, leaderless civictech movement in Taiwan) has successfully used the bridge approach to allow people to use Line, Slack and IRC, among others. They used it to migrate most of community from IRC and Line into Slack (whether that is a good direction or bad is up for debate)

Also, fodder for a comparison effort, if we feel like reticketing that: https://github.com/Kickball/awesome-selfhosted#custom
